src/gsy_e/models/strategy/forward/__init__.py

```diff
@@ -25,10 +28,26 @@ def __init__(self,
             100.0)
         super().__init__()
         self.enabled = True
```

Comment:
Can you please explain the reason for this switch? I do not see this being toggled in any part of the code. AFAIU, you enable and disable the trading with `enable_trading` and `disable_trading` events. Why is this switch needed?

Reply:
This is still a remnant from the old "disable" toggle in the UI. I do not think that the UI will need it in the future, so I will remove it, thanks!